examples/basic/main.tf

```diff
@@ -9,14 +9,4 @@ module "trusted_profile" {
     roles              = ["Viewer"]
     account_management = true
   }]
-  trusted_profile_claim_rules = [{
```

Comment:
the readme for this example needs to be updated since you are no longer creating a claim rule